### ISSUE TITLE
It is not possible to install plugins with IPM

### ISSUE THREAD

**Issue by RoySchoep** [inkdropapp/releases#1]
In version 2.1.1 plugins can't be installed on the OSX version.

```
» ipm install markdown-flowchart                                                                                                                                                          

module.js:327
    throw err;
    ^

Error: Cannot find module 'node-gyp/bin/node-gyp'
    at Function.Module._resolveFilename (module.js:325:15)
    at Function.require.resolve (internal/module.js:16:19)
    at new Install (/Applications/Inkdrop.app/Contents/Resources/app/ipm/lib/install.js:53:38)
    at Object.module.exports.run (/Applications/Inkdrop.app/Contents/Resources/app/ipm/lib/apm-cli.js:226:18)
    at Object.<anonymous> (/Applications/Inkdrop.app/Contents/Resources/app/ipm/lib/cli.js:8:7)
    at Object.<anonymous> (/Applications/Inkdrop.app/Contents/Resources/app/ipm/lib/cli.js:12:4)
    at Module._compile (module.js:409:26)
    at Object.Module._extensions..js (module.js:416:10)
    at Module.load (module.js:343:32)
    at Function.Module._load (module.js:300:12)
```

The following was the solution for me:

```
cd /Applications/Inkdrop.app/Contents/Resources/app/ipm
./bin/npm install node-gyp
```

Node-gyp is not a dependency according to the `package.json`. Adding it to the `package.json` file and running `./bin/npm install` should fix it.

**Comment by noradaiko** [comment]
Thanks for reporting.
`markdown-flowchart` is not available yet on ipm though, I'll check it soon!

**Comment by noradaiko** [comment]
I could reproduce the error. It will be fixed in the next version. Thanks!

**Comment by noradaiko** [comment]
Fixed in v2.2.0.
